scripts/setup-systemd

```diff
@@ -16,4 +16,25 @@ Restart=always
 [Install]
 WantedBy=multi-user.target" > /etc/systemd/system/grist.service
 
+echo "[Unit]
+Description=Upgrade Docker Compose services for Grist
+After=network.target docker.service
+Requires=docker.service
+
+[Service]
+Type=oneshot
+WorkingDirectory=/home/grist
+ExecStart=/home/grist/bin/upgrade-grist"  > /etc/systemd/system/grist-upgrade.service
+
+echo "[Unit]
+Description=Run Docker Compose upgrade for Grist weekly
+
+[Timer]
+OnCalendar=Sun *-*-* 00:00:00
+RandomizedDelaySec=2h
```

Comment:
What's the thinking for having a delay here? My gut instinct is that a more predictable restart time would be useful for people working on things at that time.

Reply:
It's a systemd recommendation so that if multiple services specify `weekly` on their `OnCalendar` time you can avoid a thundering herd.

Reply:
I think I'd personally prefer to prioritize predictability for the end user in this instance, rather than avoiding resource contention with other services that might be started here? Especially since that contention feels very low risk at this specific time.

Reply:
Okay, I've removed the randomness.